src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -1088,27 +1088,27 @@ namespace AirLoopHVACDOAS {
             // Summer design day
             if (DesDayInput(i).DayType == summerDesignDayTypeIndex && SummerDesignDayFlag) {
                 this->SizingCoolOATemp = DesDayInput(i).MaxDryBulb;
-                if (DesDayInput(i).HumIndType == 0) { // wet bulb temperature
+                if (DesDayInput(i).HumIndType == WeatherManager::DDHumIndType_WetBulb) { // wet bulb temperature
                     this->SizingCoolOAHumRat =
                         Psychrometrics::PsyWFnTdbTwbPb(this->SizingCoolOATemp, DesDayInput(i).HumIndValue, DataEnvironment::StdPressureSeaLevel);
-                } else if (DesDayInput(i).HumIndType == 1) { // dewpoint
+                } else if (DesDayInput(i).HumIndType == WeatherManager::DDHumIndType_DewPoint) { // dewpoint
                     this->SizingCoolOAHumRat = Psychrometrics::PsyWFnTdpPb(DesDayInput(i).HumIndValue, DataEnvironment::StdPressureSeaLevel);
-                } else if (DesDayInput(i).HumIndType == 3) {
+                } else if (DesDayInput(i).HumIndType == WeatherManager::DDHumIndType_HumRatio) {
                     this->SizingCoolOAHumRat = DesDayInput(i).HumIndValue;
-                }
+                } // else { // What about other cases?
```

Comment:
This is a TODO: I noticed that it only checks 3 values out of the  8 possible `HumIndType`